Modelica/Fluid/Examples/Tanks.mo

```diff
@@ -249,7 +249,7 @@ simulation accuracy could be increased in order to avoid errors.
       height=1.1,
       portsData={Modelica.Fluid.Vessels.BaseClasses.VesselPortsData(diameter=
           0.1, height=0.5)},
-      level_start=1.0e-10)
+      level_start=1.0e-3)
```

Comment:
Should it be the same constant?

Reply:
I think we neither need to make them constant, nor make them the same.
Just because there's an outlet just above the bottom doesn't mean that it has to start with exactly that level.

Reply:
@HansOlsson, @beutlich, this is the start level of tank2, which is nearly empty at initialization. The fact that it is the same as the height of the bottom port of tank 1 (i.e., the final value of the level of the other tank _at the end of the transient_) is totally irrelevant. I have two tank 1 m tall, so "small level" can be 1 mm for both tanks. What's the problem with that?

Reply:
I agree; I was just trying to explain what I think the comment meant.

Reply:
Yes, two constants are even clearer. In the new explanation text, 1 mm is mentioned multiple times. It would be helpful to have a reference from text to the actual named parameter.

Reply:
> Should it be the same constant?

Not necessarily. But there is also no particular reason to make then different. That's really no big deal

Reply:
> Yes, two constants are even clearer. In the new explanation text, 1 mm is mentioned multiple times. It would be helpful to have a reference from text to the actual named parameter.

I don't really think this is necessary. The two values could be the same or different, 1 mm, 0.1 mm, 2 mm, it doesn't really matter. They should be small enough, otherwise the first tank won't be really empty at the end and the first won't be really empty at the beginning. I just changed 1e-10 to 1e-3 because 1e-10 seemed to me just too small, and potentially dangerous from a numerical point of view.